### PR TITLE
improvement: trim hover body

### PR DESCRIPTION
**Pull request by kasiaMarek**
resolves: https://github.com/scalameta/metals/issues/5679

### CODE REVIEW

File: mtags-shared/src/main/scala/scala/meta/internal/pc/HoverMarkup.scala [hunk 1]
```diff
@@ -2,6 +2,8 @@ package scala.meta.internal.pc
 
 object HoverMarkup {
 
+  private val MaxHoverBodyLength = 50000
```
Comment (tgodzik) [inline mtags-shared/src/main/scala/scala/meta/internal/pc/HoverMarkup.scala:5]: Is that based on some VS Code standard? If so could you add a link to where you found it?

**Review by tgodzik — APPROVED** [review]
LGTM! Just one small comment

File: mtags-shared/src/main/scala/scala/meta/internal/pc/HoverMarkup.scala (hunk 1 above)
Comment (kasiaMarek) [reply mtags-shared/src/main/scala/scala/meta/internal/pc/HoverMarkup.scala:5]: I didn't find any information about this in docs. It's based solely on experience, that the limit is around 100000.

Comment (tgodzik) [reply mtags-shared/src/main/scala/scala/meta/internal/pc/HoverMarkup.scala:5]: Could you add a comment about that then? That it's based on the experiment as of day x.y.z

**Pull request merged**